tests/reverse.test.js

```diff
@@ -0,0 +1,9 @@
+const reverseString = require('../functions/reverse');
+
+test('Expect that this test reverses string', () => {
+  expect(reverseString('BRAZIL')).toBe('LIZARB');
```

Comment:
Hey Mike! I think you might want to use AAA pattern more clearly for this exercise. For example this can be written as 
test('Expect that this test reverses string', () => {

  const str = 'BRAZIL' // Arrange
  const ans = reverseString(str) // Act
  expect(ans).toBe('LIZARB'); // Assert

Reply:
Thanks for the reminder to use the AAA pattern.